index.html

```diff
@@ -26,46 +26,53 @@ <h1 id="" class="">Las paisas</h1>
 		</article>
 	</section>
 	<section class="carrousel-section">
-		<div class="info">
-    <h1 class="info-title">Comodidades</h1>
-    <hr class="info-line">
-      <p class="info-description">
+		<div class="container">
+		  <div class="info">
+        <h1 class="info-title">Comodidades</h1>
+        <hr class="info-line">
+        <p class="info-description">
         Nuestro hostal te ofrece todas las comodidades para que vivas la mejor experiencia.
-      </p>
-    </div>
-    <div id="carrouselContainer" class="carrousel-container">
-      <div class="carrousel">
-        <img src="assets/images/IMG_8056.JPG" class="carrousel__img"/>
-        <img src="assets/icons/arrow.svg" class="carrousel__arrow carrousel__arrow--left" data-direction="back"/>
-        <img src="assets/icons/arrow.svg" class="carrousel__arrow carrousel__arrow--right" data-direction="ahead"/>
+        </p>
       </div>
-      <div class="pointer-list" id="divBtn">
-        <div class="pointer-list__oval pointer-list__oval--gray" id="divBtn1" ></div>
-        <div class="pointer-list__oval pointer-list__oval--gray" id="divBtn2" ></div>
-        <div class="pointer-list__oval pointer-list__oval--gray" id="divBtn3" ></div>
-        <div class="pointer-list__oval pointer-list__oval--gray" id="divBtn4" ></div>
+      <div id="carrouselContainer" class="carrousel-container">
+        <div class="carrousel">
+          <img src="assets/images/IMG_8056.JPG" class="carrousel__img"/>
+          <img src="assets/icons/arrow.svg" class="carrousel__arrow carrousel__arrow--left" data-direction="back"/>
+          <img src="assets/icons/arrow.svg" class="carrousel__arrow carrousel__arrow--right" data-direction="ahead"/>
```

Comment:
I think that the images shouldn't be in the HTML, 
it's better if they are assigned in the CSS.
Please, If I'm wrong please correct me.

Reply:
That part is now in develop branch, this pr is only for center carrousel and center botton galery and that little points.

Reply:
@hanna446 your are right but that should be addressed in a refactor to the carrousel in the future.